install_script.sh.template

```diff
@@ -378,6 +378,214 @@ function ensure_config_file_exists() {
+    json_logs="["
+    while IFS= read -r line; do
+        if [ -z "$line" ]; then
+            continue
+        fi
+        json_logs+="{\"message\": \"$line\", \"level\": \"ERROR\", \"trace_id\": \"${trace_id}\", \"span_id\": \"${trace_id}\"},"
+    done <<< "$log_lines_stderr"
+    while IFS= read -r line;
+    do
+        if [ -z "$line" ]; then
+            continue
+        fi
+        json_logs+="{\"message\": \"$line\", \"level\": \"DEBUG\", \"trace_id\": \"${trace_id}\", \"span_id\": \"${trace_id}\"},"
+    done <<< "$log_lines_stdout"
+    json_logs="${json_logs%,}]"
+
+    telemetry_logs=$(cat <<-END
+    {
+        "api_version": "v2",
+        "request_type": "logs",
+        "tracer_time": ${time_now_seconds},
+        "runtime_id": "${trace_id}",
+        "seq_id": 2,
+        "origin": "linux-install-script",
+        "host": {
+            "hostname": "$(uname -n)",
+            "os": "${os}",
+            "distribution": "${distribution}",
+            "architecture": "$(uname -m)",
+            "kernel_version": "$(uname -v)",
+            "kernel_name": "$(uname -s)",
+            "kernel_release": "$(uname -r)"
+        },
+        "application": {
+            "service_name": "datadog-linux-install-script",
+            "service_version": "${install_script_version}",
+            "language_name": "UNKNOWN",
+            "language_version": "n/a",
+            "tracer_version": "n/a"
+        },
+        "payload": {
+            "logs": ${json_logs}
+        }
+    }
+END
+)
+
+    if ! echo "$telemetry_logs" | curl -f -sSL --retry 5 -o /dev/null \
+        "$telemetry_url" \
+        --header 'Content-Type: application/json' \
+        --header "DD-Api-Key: $apikey" \
+        --data @-
+    then
+        echo "Unable to send logs telemetry\n"
+    fi
+
+    if ! echo "$telemetry_trace" | curl -f -sSL --retry 5 -o /dev/null \
+    "$telemetry_url" \
+    --header 'Content-Type: application/json' \
+    --header "DD-Api-Key: $apikey" \
+    --data @-
+    then
+        echo "Unable to send trace telemetry\n"
+    fi
+}
+
+function configure_installer() {
+    local sudo_cmd="$1"
+    local trace_id="$2"
+    local apm_instrumentation_enabled="$3"
+
+    $sudo_cmd mkdir -p /etc/datadog-agent
+    $sudo_cmd bash -c "cat <<EOT > /etc/datadog-agent/datadog-installer.ini
+[features]
+apm_instrumentation=$apm_instrumentation_enabled
+
+[telemetry]
+trace_id=$trace_id
+parent_id=$trace_id
+priority=2
+EOT"
+}
+
+function _install_installer() {
+    local sudo_cmd="$1"
+    local os="$2"
+    local distribution="$3"
+    local apm_instrumentation_enabled="$4"
+
+    if [ "$os" == "Debian" ] && [ "$distribution" == "Debian" ] && [ -n "$apm_instrumentation_enabled" ]; then
+        echo -e "\033[34m\n* Installing the Datadog installer\n\033[0m"
+        configure_installer "$sudo_cmd" "$trace_id" "$apm_instrumentation_enabled"
+        $sudo_cmd apt-get install -o Acquire::Retries="5" -y --force-yes "datadog-installer"
```

Comment:
is `--force-yes` really needed? It's deprecated and causes a bunch of warnings to be emitted during each test

Reply:
I copy / pasted what we had for the other DEB packages, I don't actually know if we need this, it creates the same logs for the regular agent install 😅

Reply:
It probably should be removed from there as well :)

Reply:
Ok I'll remove it here and we can use this as a test that it's safe to remove :trollface:

Reply:
Hm actually I might have been too eager to remove it, it's not supported by ubuntu 14.04, which we still support :(

Reply:
classic, adding it back lol

Reply:
Sorry :sweat_smile: